pennylane/templates/subroutines/out_multiplier.py

```diff
@@ -208,16 +216,6 @@ def map_wires(self, wire_map: dict):
             new_dict["work_wires"],
         )
 
-    @property
-    def wires(self):
-        """All wires involved in the operation."""
-        return (
-            self.hyperparameters["x_wires"]
-            + self.hyperparameters["y_wires"]
-            + self.hyperparameters["output_wires"]
-            + self.hyperparameters["work_wires"]
-        )
```

Comment:
Yes, normally we just use the init method of the base class to set this property. There would have to be a special reason to override it here. Was there a specific reason this was added in the first place? If not its safe to remove 👍🏼

Reply:
It was added recently and I do not remember any discussion about it